docs/how-tos/validate-nexus-file.md

```diff
@@ -0,0 +1,802 @@
+
+Then use:
+
+```
+read_nexus -f C:\nexusvalidation\Raman.nxs
+```
+
+shows the output like this:
+
+```
+===== FIELD (//entry/data/spectrum_data_y): <HDF5 dataset "spectrum_data_y": shape (1600,), type "<f8">
+DEBUG: ===== FIELD (//entry/data/spectrum_data_y): <HDF5 dataset "spectrum_data_y": shape (1600,), type "<f8">
+value: [ 288.5499878  289.         288.4500122 ... 1875.        1889.349976 ...
+DEBUG: value: [ 288.5499878  289.         288.4500122 ... 1875.        1889.349976 ...
+Dataset referenced as NXdata SIGNAL
+DEBUG: Dataset referenced as NXdata SIGNAL
+===== ATTRS (//entry/data/spectrum_data_y@long_name)
+DEBUG: ===== ATTRS (//entry/data/spectrum_data_y@long_name)
+value: Raman Intensity 
+DEBUG: value: Raman Intensity 
+Dataset referenced as NXdata SIGNAL
+DEBUG: Dataset referenced as NXdata SIGNAL
+===== ATTRS (//entry/data/spectrum_data_y@units)
+DEBUG: ===== ATTRS (//entry/data/spectrum_data_y@units)
+value: counts 
+DEBUG: value: counts 
+
+DEBUG: 
+For Axis #0, 1 axes have been identified: [<HDF5 dataset "spectrum_data_x_Raman": shape (1600,), type "<f8">]
+DEBUG: For Axis #0, 1 axes have been identified: [<HDF5 dataset "spectrum_data_x_Raman": shape (1600,), type "<f8">]
+```
+
+Search for fields which are not found in the NeXus definition by searching for the line: "DEBUG: NOT IN SCHEMA". Recheck the used NeXus definition to eliminate the problem. Be careful with upper and lower case notation and correct spelling.
+
+Keep in mind, that the output provides quite some information. This is useful for software development, but may be a bit too much for validation purposes.
+
+Similar features as the tables and messages provided from punx and cnxvalidate for pynxtools are [planned to be implemented in the future](<https://github.com/FAIRmat-NFDI/pynxtools/pull/333>).
+
+
+
+
+
+
+
+
+
+
+# 2. cnxvalidate
+
+This package is written in C. It is allows a command line evocation like:
+
+```
+nxvalidate -l appdefdir datafile
+```
+
+1. nxvalidate: calls the software function
+
+2. \-l appdefdir: points to the location of the NeXus definitions you want to use. This is a path to a folder called "defintions".
+
+3. datafile: This is the path to the .nxs file which should be checked.
+
+This output shows warnings like:
+
+```
+definition=NXoptical_spectroscopy.nxdl.xml message="Required attribute URL missing" nxdlPath=/NXentry/definition sev=error dataPath=/entry/definition dataFile=NXopt_minimal_example.nxs
+```
+
+and indicates the entry of the .nxs file, which is incorrect and what the respective problem is. It also points to the NeXus definition (.nxdl.xml file), in which this conflict was found.
+
+# 2.1. Installation (Linux only)
+
+This [did not work for me on windows](/installation_notes_nxvalidate.md) (The problem was the software cmake couldn't fine the libxml2 library. Though, if you solve this, this maybe work on windows).
```

Comment:
Remove the personal reference "me".

Maybe:
"Note: You can find more information about installing nxvalidate [here](installation_notes_nxvalidate.md). Note that installation on Windows can be tricky because cmake can sometimes not find the libxml2 library. Though, if you solve this, this maybe work on windows)."

Reply:
used it as suggested.